src/components/Repo/Repo.js

```diff
@@ -0,0 +1,35 @@
+import React, { Fragment } from 'react'
+import styled from 'styled-components'
+import BodyStylised from '../Typography/BodyStylised'
+import remcalc from 'remcalc'
+import Subtitle from '../Typography/Subtitle'
+import Hr from '../Common/Hr'
+
+const StyledHr = styled(Hr)`
+  margin-top: ${remcalc(30)};
+`
+
+const Repo = ({
+  url,
+  nameWithOwner,
+  pullRequestCount,
+  starCount,
+  small = false
+}) => (
+  <Fragment>
```

Comment:
There should be `12px` padding on the top of this component:
![image](https://user-images.githubusercontent.com/15656538/58709039-22ee3f80-83b1-11e9-98e1-44f3309652b6.png)

Could potentially be achieved by having `noPaddingBottom` on `Subtitle` rather than `noPadding`?

Reply:
updated